epoxy-adapter/src/main/java/com/airbnb/epoxy/DiffHelper.java

```diff
@@ -17,18 +14,29 @@
  * Helper to track changes in the models list.
  */
 class DiffHelper {
-  private static final boolean USE_DIFF_UTIL = false;
```

Comment:
I also removed the optional usage of the support library diff util since it is definitely slower now, and keeping it around made the code messier and more complex